static/index.html

```diff
@@ -2,23 +2,25 @@
 <head>
 <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui" />
 <link rel="stylesheet" href="leaflet.css" />
-<script src="leaflet.js"></script>
+<link rel="stylesheet" href="css/button.css"/>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.min.js"></script>
+    <script src="leaflet.js"></script>
 <style type="text/css">
 body, #map {
   height: 100%;
 }
 </style>
 </head>
 <body>
+<div id="header" style="width: 100%;">
```

Comment:
Might be even easier to use built-in layer selector:
http://leafletjs.com/examples/layers-control/